modules/local/fetchtool_reads.nf

```diff
@@ -3,7 +3,7 @@ process FETCHTOOL_READS {
 
```

Comment:
I think that module should be moved into folder fetchtool_reads and renamed into main.nf for consistency of nf-core structure.

Reply:
Well.. I'm not a fan of `local/<tool>/main.nf` as it creates loads of folder with no benefit. The modules you install from nf-modules/nf-core having that structure makes sense.

I think we should define what is the team standard, I vote for not having that and just using the tool name.nf.

Reply:
The nf-core pipelines don't have the extra module folders -> https://github.com/nf-core/mag/tree/master/modules/local

Reply:
It doesn't matter for what we vote if nf-core will change it one day to be strict stardard :) 
This is the only one thing that I concern. I'm following nf-core modules repo structure.